responsibleai/responsibleai/modelanalysis/model_analysis.py

```diff
@@ -309,6 +309,13 @@ def _validate_model_analysis_input_parameters(
                             ' getting predictions via predict_proba()'
                         )
 
+                if task_type == ModelTask.REGRESSION:
+                    if hasattr(model, 'predict_proba'):
```

Comment:
can you use the constant from here:

https://github.com/microsoft/responsible-ai-widgets/blob/850bf429f20b5401b7907e249d219dbd5d97f6c9/responsibleai/responsibleai/_internal/constants.py#L56